tests/delivery-interfaces.test.js

```diff
@@ -0,0 +1,64 @@
+import {
+  setupDatabase,
+  createLoggedInGraphqlFetch,
+  createAnonymousGraphqlFetch,
+} from "./helpers";
+import { ADMIN_TOKEN } from "./seeds/users";
+
+let connection;
+let graphqlFetch;
+
+describe("DeliveryInterfaces", () => {
+  beforeAll(async () => {
+    [, connection] = await setupDatabase();
+    graphqlFetch = await createLoggedInGraphqlFetch(ADMIN_TOKEN);
+  });
+
+  afterAll(async () => {
+    await connection.close();
+  });
+
+  describe("For logged in users", () => {
```

Comment:
If this only works for admin, and I guess it does, then it should say "For administrators" or at least "Users with privileges" or something